### PR TITLE
Win64: Get rid of 2 obsolete fields in TypeInfo_Struct

### PR DESCRIPTION
**Pull request by kinke**
The required changes to druntime turned out a bit more extensive than I had hoped for at the beginning, but since the job is already done, I thought I'd just open a PR.

### CODE REVIEW

File: gen/typinf.cpp [hunk 1]
```diff
@@ -515,7 +514,7 @@ class LLVMDefineVisitor : public Visitor {
     // uint m_align;
     b.push_uint(DtoAlignment(tc));
 
-    if (global.params.is64bit) {
+    if (isSystemVx64) {
```
Comment (kinke) [inline gen/typinf.cpp]: I'm rather suprised this hasn't bugged anyone until now. In druntime, the fields were only there for `version (X86_64)`, so this must have been broken for all other 64-bit targets.

**Comment by kinke** [comment]
@rainers: Do you think it'd make sense to upstream the druntime portion?

**Comment by rainers** [comment]
> @rainers: Do you think it'd make sense to upstream the druntime portion?

Took a while until I found the actual removed members because they don't show up in the diffs.IMO it's unlikely worth the trouble to change upstream dmd and druntime. Unsure about removing them in LDC as it just increases the differences...

**Comment by kinke** [comment]
This has happened as of LDC v1.22.

**Pull request closed without merging**